store-implementation/parquet-store/src/main/java/uk/gov/gchq/gaffer/parquetstore/partitioner/GroupPartitioner.java

```diff
@@ -87,16 +89,17 @@ public int getPartitionId(final PartitionKey partitionKey) {
      * @return a list of all the partition ids which could contain data matching the provided partial key
      */
     public List<Integer> getPartitionIds(final Object[] partialKey) {
+        final int partitionKeyLength = splitPoints.isEmpty() ? 0 : splitPoints.get(0).getLength();
```

Comment:
If this is moved below the if(0 == splitPoints.size()) (to be changed in below comment) then it can just be set with ``` final int partitionKetLength = splitPoints.get(0).getLength();``` as we know it will not be empty because we have checked above